source/classes.tex

```diff
@@ -6555,7 +6555,7 @@
 \end{codeblock}
 Here the criteria for elision can
 eliminate
-the copying of the local automatic object
+the copying of the object with automatic storage duration
 \tcode{t}
```

Comment:
This makes it sound like `t` is a name for the storage duration, not for the object.

Reply:
@zygoloid Changing it to "the object `t` with automatic storage duration" would resolve this